unittests/snapshot_tests.cpp

```diff
@@ -23,7 +23,7 @@ chainbase::bfs::path get_parent_path(chainbase::bfs::path blocks_dir, int ordina
       blocks_dir = blocks_dir.parent_path();
       leaf_dir = blocks_dir.filename();
       try {
-         auto ordinal_for_config = boost::lexical_cast<int>(leaf_dir.generic_string());
+         boost::lexical_cast<int>(leaf_dir.generic_string());
```

Comment:
why not remove?

Reply:
The `lexical_cast` can throw which is why the try-catch is there.